documentation/sphinx/source/configuration.rst

```diff
@@ -708,7 +708,7 @@ The ``usable_regions`` configuration option determines the number of regions whi
 
 .. warning:: In release 6.0, ``usable_regions`` can only be configured to the values of ``1`` or ``2``, and a maximum of 2 regions can be defined in the ``regions`` json object.
 
-Increasing the ``usable_regions`` will start copying data from the active region to the remote region. Reducing the ``usable_regions`` will immediately drop the replicas in the remote region. During these changes, only one primary datacenter can have priority >= 0. This enforces exactly which region will lose its replica.
+Increasing the ``usable_regions`` will start copying data from the active region to the remote region. Reducing the ``usable_regions`` will drop the replicas in the remote region. During these changes, only one primary datacenter can have priority >= 0. This enforces exactly which region will lose its replica.
```

Comment:
@jzhou77 - do you think this change is good enough? Thanks!

Reply:
Can we add more explanation of what happened after the configuration change? Something like:

```
Reducing the ``usable_regions`` will drop the replicas in the remote region: data distributor will notice the change and start removing data stored in the remote region's now "undesired" storage servers.
```

Reply:
Updated. Not sure if this version is better: "Reducing the ``usable_regions`` will start removing data from the replicas in the remote region (on data distributor noticing the configuration change and marking the remote region's replicas as "undesired")".

Reply:
marking undesired is not the complete story, and we should mention removing data stored (which is essentially noop data movement).

Reply:
That's what "dropping the replicas" means, right? Will this statement capture this idea better: 

"Reducing the usable_regions will start removing data from the replicas in the remote region (on data distributor noticing the configuration change and marking the remote region's replicas as "undesired")".